tests/php/Forms/ValidatorTest/TestValidatorList.php

```diff
@@ -0,0 +1,25 @@
+<?php
```

Comment:
Could you do this with reflection instead?

Reply:
Mostly I was just following the paradigm already set by the `Validator` tests.

Reply:
OK fair enough :+1: